scripts/start-geth

```diff
@@ -24,14 +24,26 @@ $PASSWORD
 EOF
 
 # import validator key
+validator_key=$(python -c """
```

Comment:
But can we run `start-geth` directly from cli after this?

Reply:
Yes, we can, in nix-shell.

Reply:
where is the `.env` sourced?

Reply:
Or one should source it by himself, or put it in shell profile?

Reply:
Yes, in nix shell, source in `start-geth` defined in `script.nix`. For the script itself, no source.

Reply:
It is difficult to add it in script, because the `.env` relative path depends on the user's working directory.

Reply:
Or use full path, but full path also can be changed.